docs/rest_api.md

```diff
@@ -1251,3 +1251,31 @@ Remove a user from a project.
 Authorization requirements:
 
 * Administrative access.
+
+#### list_active_extensions
+
+`GET /active_extensions`
+
+Request Body:
+
+{}
+
+Response Body:
+
+[
+    [
+        "hil.ext.switches.mock",
+        ""
+    ],
+    [
+        "hil.ext.network_allocators.null",
+        ""
+    ],
+    ...
+]
```

Comment:
Rather than handing back the nested list we get from configparser, just return a list of the names of the extensions.